src/commonMain/kotlin/komem/litmus/runners/LitmusTestRunnerUtil.kt

```diff
@@ -0,0 +1,74 @@
+package komem.litmus.runners
+
+import komem.litmus.*
+import kotlin.time.Duration
+import kotlin.time.TimeSource
+
+fun LitmusTestRunner.runTest(
```

Comment:
Why you decided to move these functions into separate file? 
It is perfectly fine to place the extensions functions in the same file where the interface itself is defined (`LitmusTestRunner.kt` in this case).